frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/InteractiveEmbeddingOptionCard/InteractiveEmbeddingOptionCard.tsx

```diff
@@ -29,16 +36,15 @@ export const InteractiveEmbeddingOptionCard = ({
   const quickStartUrl = useSelector(state =>
     getDocsUrl(state, {
       page: "embedding/interactive-embedding-quick-start-guide",
+      utm: interactiveEmbeddingUtmTags,
     }),
   );
 
-  const quickStartLinkWithUtm = `${quickStartUrl}?utm_source=${plan}&utm_media=embed-settings`;
-
   return (
     <EmbeddingOption
       icon={
         <InteractiveEmbeddingIcon
-          disabled={isEE && !isInteractiveEmbeddingEnabled}
+          disabled={!isEE || !isInteractiveEmbeddingEnabled}
```

Comment:
I thought `isInteractiveEmbeddingEnabled` was going to be false on OSS, but if you have this value set to `true` it will stay true on OSS.

This condition could be understood more easily if we flip the logic. We should only show _enable_ icon when `isEE && isInteractiveEmbeddingEnabled`. So the negation of that is `!isEE || !isInteractiveEmbeddingEnabled`

Reply:
should that be something we should sort out on the BE? We shouldn't be enabling that flag unless the user is EE

Reply:
It's fine for now but just a thought for the future

Reply:
Yeah, this should be BE's responsibility for sure.